lib/awful/completion.lua

```diff
@@ -154,10 +161,25 @@ function completion.shell(command, cur_pos, ncomp, shell)
         while true do
```

Comment:
Huh, I accidently mentioned a problem that incorrectly breaks the tests. If there is a command `truecrypt` (or `true_foobar`, or `true_*` ...) in `PATH`, then most (if not all) tests regarding `true` will fail, because you assume to be just one completion item, namely `true`. It isn't directly related with the current PR, so I'll start another one for that (as soon as I have found out how to deal with it, maybe sublist matching instead of list equality?).